### PR TITLE
(1946) IATI export iteration

### PR DESCRIPTION
**Pull request by pezholio**
This PR moves the location of the XML exports from the Organisation view page to an Organisation's export page. It also moves the generation of the XML to the exports controller and cleans up/rationalises a bunch of things (including the XML tests, which were quite slow)

## Screenshots

image

### CODE REVIEW

**Comment by pezholio** [comment]
[IATI export iteration](https://trello.com/c/yQjzugEk/1946-iati-export-iteration)

**Comment by mec** [comment]
@pezholio i've given this a first pass, how do you feel about the following general points before I give it my all:

- align the export titles to IATI as that is the only thing they are used for. e.g. "Global Challenges Research Fund (GCRF) IATI export for level B (programme) activities" (changing the values as needed)
- is there a way to not show a export where a DP has no activities (e.g UKSA do not report on newton)?
- a changelog entry for when the UI moves

**Comment by pezholio** [comment]
Cool - all good suggestions. Will get on it 👍

**Comment by pezholio** [comment]
@mec This is ready for another looksee now

File: config/routes.rb [hunk 1]
```diff
@@ -29,6 +29,9 @@
     namespace :exports do
       resources :organisations, only: [:show] do
         get "transactions", on: :member
+        get "programme_activities", on: :member
```
Comment (mec) [inline config/routes.rb:32]: Is it worth adding 'iati' to these, jsut to be super clear and keep them in a namespace of sorts?

File: app/views/staff/organisations/show.html.haml [hunk 2]
```diff
@@ -42,36 +42,3 @@
 
       - if policy(:organisation).edit?
         = link_to t("page_content.organisation.button.edit_details"), edit_organisation_path(@organisation_presenter), class: "govuk-button"
-
```
Comment (mec) [inline app/views/staff/organisations/show.html.haml:45]: :heart:

File: .gitignore [hunk 3]
```diff
@@ -135,3 +135,5 @@ terraform/.terraform/*
 /yarn-error.log
 yarn-debug.log*
 .yarn-integrity
+
+!app/views/**/*.xml.haml
```
Comment (mec) [inline .gitignore:139]: 👍

**Review by mec — APPROVED** [review]
I have one comment about the routes, but it shouldn't block this. Sorry for the unclear ealy feedback - thanks for taking it onboard!

File: config/routes.rb (hunk 1 above)
Comment (pezholio) [reply config/routes.rb]: Good plan 👍 I've also namespaced the model, just to be consistent

**Pull request merged**